src/player.cpp

```diff
@@ -11169,6 +11176,29 @@ object_type player::get_grab_type() const
     return grab_type;
 }
 
+void player::start_hauling()
+{
+    add_msg( _( "You start hauling items along the ground." ) );
+    if ( is_armed() ) {
+        add_msg( m_warning, _( "Your hands are not free, which makes hauling slower." ) );
+    }
+    this->hauling = true;
+}
+
+void player::stop_hauling()
+{
+    add_msg( _( "You stop hauling items." ) );
+    this->hauling = false;
```

Comment:
Why are you accessing `hauling` via `this` instead of directly?

Reply:
Good  question! Python brain damage? :)